exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -142,50 +153,144 @@ def export(
     def shutdown(self) -> None:
         raise NotImplementedError()
 
-    def convert_to_timeseries(
+    def _convert_to_timeseries(
```

Comment:
Do these functions need to be part of the exporter?

Reply:
No, they really don't. We were unsure were to put them. Will follow-up on this.

Reply:
After further discussion, keeping the supported aggregator to converter method map as in the class namespace is reasonable since it is not used anywhere outside the class. Therefore, all the converter methods now call "self" and can remain as normal methods within the class. Let me know if you agree with this conclusion.